package exception;

public class Main {

	public static void main(String[] args) {
		
		
		
		
	System.out.println("Välkommen till hela detta programmet! ");
		
		
		
		
		
		try {
			
			
		String[]name = {"Anna " , "Johan"};
		
		System.out.println(name[2]);
		
		
		
	       }  catch (ArrayIndexOutOfBoundsException e){
	    	   
	    	   System.out.println("du kan inte välja en person som inte finns var vänlig försök igen: ");
	        	 
	        	 
	        	 
	        
		
		
			 } catch  (Exception e) {
				 System.out.println("du kan inte välja en person som inte finns var vänlig försök igen ");
			
		
		}
		
	

		System.out.println("hoppas du har haft det bra");
		
		
	}

}
